Directory.Build.props

```diff
@@ -257,7 +257,6 @@
     <LangVersion>latest</LangVersion>
     <LangVersion Condition="'$(Language)' == 'C#'">preview</LangVersion>
     <!-- Enables Strict mode for Roslyn compiler -->
-    <Features>strict</Features>
```

Comment:
?

Reply:
So lame. This was because I wanted to test the empty ;. 😞

Reply:
Fixed.